developerguide/provision-wo-cert.md

```diff
@@ -104,7 +104,7 @@ You must manage the trusted user's access and permission to perform this procedu
            "iot:CreateProvisioningClaim"
        ],
        "Resource": [
-           "arn:aws:aws-region:aws-account-id:provisioningtemplate/templateName"
+           "arn:aws:aws-service:aws-region:aws-account-id:provisioningtemplate/templateName"
```

Comment:
Thanks for the feedback! Think I'll go with `iot` as that seems to be used for other policies on this page.